src/Tasks/Exec.cs

```diff
@@ -617,6 +617,15 @@ protected internal override void AddCommandLineCommands(CommandLineBuilderExtens
                         batchFileForCommandLine = NativeMethodsShared.GetShortFilePath(batchFileForCommandLine);
                         batchFileForCommandLine = batchFileForCommandLine.Replace("&", "^&");
                     }
+
+                    // cmd needs parens to be escaped when executing files with the /C flag.
+                    // consider the case where the user has a parenthesis in ther username (which is uncommon, but valid)
+                    if ((batchFileForCommandLine.Contains("(") && !batchFileForCommandLine.Contains("^(")) || (batchFileForCommandLine.Contains(")") && !batchFileForCommandLine.Contains("^)")))
```

Comment:
Why aren't you just escaping all instances of `(`/`)` without preceding `^`? Seems like you'll miss some this way in, for instance, `ben(dev^(atMS^))`